### DISCUSSION TITLE
Error when running ros2_bag utility

### DISCUSSION THREAD

**Discussion by LiyouZhou** [mvukov/rules_ros2#75] [Q&A]
I created my ros2_bag utility so I can make it depend on some of my custom messages, but it cannot seem to load the sqlite3 plugin. The `chatter:bag` example works fine but not when I create my own target in my own repo. Where is the sqlite3 plugin being loaded from? does the impl assume any hard coded paths?

`BUILD`
```
ros2_bag(
    name = "bag",
    idl_deps = [
        "@ros2_common_interfaces//:std_msgs",
        "@ros2_rcl_interfaces//:rcl_interfaces",
        "@ros2_rosbag2//:rosbag2_interfaces",
    ],
)
```

```
[INFO] [1678202742.206147912] [rosbag2_recorder]: Press SPACE for pausing/resuming
[ERROR] [1678202742.207030599] [rosbag2_storage]: Unable to load plugin 'sqlite3': MultiLibraryClassLoader: Could not create class of type rosbag2_storage_plugins::SqliteStorage
[ERROR] [1678202742.207054290] [rosbag2_storage]: Could not load/open plugin with storage id 'sqlite3'
No storage could be initialized. Abort
```

**Comment by mvukov** [comment]
> Where is the sqlite3 plugin being loaded from? does the impl assume any hard coded paths?

No, there are no hard-coded paths. Can you create a minimum sample repo to demonstrate the problem?

**Reply by mvukov** [reply]
BTW, might be related to https://github.com/mvukov/rules_ros2/issues/78.
